src/neuroconv/datainterfaces/text/excel/requirements.txt

```diff
@@ -0,0 +1,3 @@
+openpyxl
+xlrd
```

Comment:
What are these two for?

Reply:
pd.read_excel requires openpyxl for .xslx files and xlrd for .xls files